controllers/component_build_controller_pac.go

```diff
@@ -869,6 +871,13 @@ func generatePaCPipelineRunForComponent(
 		{Name: "revision", Value: tektonapi.ArrayOrString{Type: "string", StringVal: "{{revision}}"}},
 		{Name: "output-image", Value: tektonapi.ArrayOrString{Type: "string", StringVal: proposedImage}},
 	}
+	if onPull {
+		expiration := os.Getenv(pipelineRunOnPRExpirationEnvVar)
```

Comment:
should we validate the value provided via env var? i.e. if the value is invalid, log that information and use the default one?

Reply:
added